lib/clients/AwsClient.py

```diff
@@ -300,11 +300,17 @@ def _delete_volume(self, volume_id):
             self.logger.error(message)
             raise Exception(message)
 
+    def _refresh_devices_list(self,instance_id):
```

Comment:
is it good to rename ==> `_refresh_attached_devices_list()` ?? Just a thought. Otherwise LGTM

Reply:
Will do. Thanks.